client/src/app/shared/components/header/header.component.scss

```diff
@@ -0,0 +1,8 @@
+nav.top-navbar-container{
+   background-color: #e6e5e5f5; 
+   height: 72px;
+   & u.ml-auto {
+        margin-left: auto !important;
```

Comment:
Do not use `!import` anywhere in the app.
Also, do not use classes like `ml-auto` etc. 
Give the item a class that suits the item. Like `.user-list`, `.user-details` etc, use that class here.